File: RollTheDice/Assets/_Project/API/Model/DTO/GameDTO/MoneyDTO/ValueDTO.cs
using UnityEngine;
using System.Collections;

namespace Assets._Project.API.Model.DTO.GameDTO.MoneyDTO
{
	public class ValueDTO 
	{
		public  double Amount { get; set; }
		public long? CurrencyId { get; set; }

		public ValueDTO(double amount, long currencyId)
		{
			Amount = amount;
            CurrencyId = currencyId;
        }

        public ValueDTO()
        {
     
        }
    }
}

File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/Model/DTO/GameDTO/MoneyDTO/ValueDTO.java
package com.example.APIRollTheDice.Model.DTO.GameDTO.MoneyDTO;

import com.fasterxml.jackson.annotation.JsonProperty;

public class ValueDTO {
    @JsonProperty("Amount")
    private double amount;
    @JsonProperty("CurrencyId")
    private Long currencyId;

    public ValueDTO() {
    }

    public ValueDTO(double amount, Long currencyId) {
        this.amount = amount;
        this.currencyId = currencyId;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public Long getCurrencyId() {
        return currencyId;
    }

    public void setCurrencyId(Long currencyId) {
        this.currencyId = currencyId;
    }
}
